index.bs

```diff
@@ -2198,6 +2198,9 @@ The following abstract operations operate on {{ReadableStream}} instances at a h
     |originalError|, then:
    1. If |shuttingDown| is true, abort these substeps.
    1. Set |shuttingDown| to true.
+   1. If |reader| [=implements=] {{ReadableStreamBYOBReader}}, perform
+      ! [$ReadableStreamBYOBReaderRelease$](|reader|).
+   1. Otherwise, perform ! [$ReadableStreamDefaultReaderRelease$](|reader|).
```

Comment:
Drive-by fix: the user agent is allowed to use a BYOB reader for the pipe (see step 8), but now we have *separate* `Release()` abstract ops for default and BYOB readers. (Before #1168, we had a single `ReadableStreamGenericReaderRelease` that worked for both.)

Reply:
Mind factoring this out into a separate PR?

Reply:
Done, see #1209.